src/Azure.Functions.Cli/Actions/AzureActions/BaseAzureAction.cs

```diff
@@ -14,6 +14,31 @@ namespace Azure.Functions.Cli.Actions.AzureActions
 {
     abstract class BaseAzureAction : BaseAction, IInitializableAction
     {
+        private const string _getAccessTokenPowerShellScript = @"
```

Comment:
This variable is not used anywhere now.

Reply:
good catch... omnisharp on linux should have warned me about that...

Reply:
(fixed)